web/libs/datamanager/src/components/Common/RadioGroup/RadioGroup.scss

```
@@ -0,0 +1,114 @@
+.radio-group {
+  --radius: 8px;
+  --height: 40px;
+  --padding: 4px;
+  --font-size: 16px;
+  --button-padding: 0 10px;
+  --button-checked-shadow: 0px 1px 0px rgba(0, 0, 0, 0.1), 0px 0px 0px 1px rgba(0, 0, 0, 0.02), 0px 5px 10px rgba(0, 0, 0, 0.15);
+
+  height: var(--height);
+  border-radius: var(--radius);
+  padding: var(--padding);
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0.05), rgba(0, 0, 0, 0.05)), #FFFFFF;
+  box-shadow: inset 0px 1px 0px rgba(0, 0, 0, 0.05), inset 0px 0px 0px 1px rgba(0, 0, 0, 0.05);
```

Flagging as opportunity for css var replacement. Further usage review is needed.
```suggestion
  background: linear-gradient(0deg, rgba(var(--black), 0.05), rgba(var(--black), 0.05)), #FFFFFF;
  box-shadow: inset 0px 1px 0px rgba(var(--black), 0.05), inset 0px 0px 0px 1px rgba(var(--black), 0.05);
```